templates/index-bare.html

```diff
@@ -0,0 +1 @@
+<!DOCTYPE html><html lang="en"><head></head><body><p></p><br/></body></html>
```

Comment:
The tests are failing on windows due this file 

```
<!DOCTYPE html><html lang=\"en\"><head></head><body><p></p><br/></body></html>\r\n
```

Reply:
Should be fixed.